src/app/loadout/known-values.ts

```diff
@@ -27,7 +27,7 @@ export const raidPlugCategoryHashes = [
   PlugCategoryHashes.EnhancementsSeasonOutlaw, // last wish
   PlugCategoryHashes.EnhancementsRaidGarden, // garden of salvation
   PlugCategoryHashes.EnhancementsRaidDescent, // deep stone crypt
-  PlugCategoryHashes.EnhancementsSeasonMaverick, // nightmare
+  PlugCategoryHashes.EnhancementsRaidV520, // vault of glass
```

Comment:
Not sure why nightmare was in this but it shoundn't have been.

Also I am thinking we can start generating this by looking for the word raid in the `plugCategoryIdentifier `. There will need to just be a special case for the lastwish one as it doesn't contain it. Thoughts?